backend/src/flotilla/database/crud.py

```diff
@@ -105,7 +74,7 @@ def create_event(
 
 
 def remove_event(db: Session, event_id: int) -> None:
```

Comment:
Not this PR, but this should be `delete_event` to follow CRUD name convention

Reply:
This function is named `remove_event` because it is used in another function called `delete_event`